<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowSelfTest</class>
 <widget class="QMainWindow" name="MainWindowSelfTest">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1331</width>
      <height>721</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>100</number>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>26</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>27</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>28</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>29</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>31</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>32</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>33</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>34</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>35</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>36</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>37</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>38</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>39</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>40</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>41</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>42</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>43</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>44</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>45</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>46</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>47</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>48</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>49</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>50</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>51</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>52</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>53</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>54</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>55</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>56</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>57</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>58</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>59</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>60</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>61</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>62</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>63</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>64</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>65</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>66</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>67</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>68</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>69</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>70</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>71</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>72</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>73</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>74</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>75</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>76</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>77</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>78</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>79</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>80</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>81</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>82</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>83</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>84</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>85</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>86</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>87</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>88</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>89</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>90</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>91</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>92</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>93</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>94</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>95</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>96</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>97</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>98</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>99</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>100</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>运行时间(分钟)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇14</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇15</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇16</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇17</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇18</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇19</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风扇20</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>水泵</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TEC1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TEC2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TEC3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TEC4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TEC5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TEC6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电流1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电流2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电流3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电流4</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>740</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>保存自动测试设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>740</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1353</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
